corehq/apps/export/views/new.py

```diff
@@ -540,5 +541,10 @@ def get(self, request, domain, export_id, *args, **kwargs):
                 request,
                 format_html(_("Export <strong>{}</strong> created."), new_export.name)
             )
-        redirect = request.GET.get('next', reverse('data_interfaces_default', args=[domain]))
-        return HttpResponseRedirect(redirect)
+        redirect = request.GET.get('next')
+        if redirect:
+            is_valid = url_has_allowed_host_and_scheme(redirect, allowed_hosts=None)
```

Comment:
Is the idea with `allowed_hosts=None` that the supplied `redirect` must always be a relative path or URL specifying no domain?

In other words, I think this will fail with a URL like `https://www.commcarehq.org/a/login/?next=https://www.commcarehq.org/a/some-domain/dashboard`